src/Microsoft.DotNet.XHarness.iOS/AppRunner.cs

```diff
@@ -102,6 +104,26 @@ public class AppRunner
                 new SetEnvVariableArgument(EnviromentVariables.DisableSystemPermissionTests, 1),
             };
 
+            if ((skippedMethods != null && skippedMethods.Length > 0) ||
+                (skippedTestClasses != null && skippedTestClasses.Length > 0))
```

Comment:
Not sure this is any improvement, but:

```suggestion
            if (skippedMethods?.Any() ?? skippedTestClasses?.Any() ?? false)
```